hw/ip/otp_ctrl/data/BUILD

```diff
@@ -232,6 +232,13 @@ otp_json(
                 ),
                 # By default, ROM_EXT's bootstrap feature should be disabled.
                 "OWNER_SW_CFG_ROM_EXT_BOOTSTRAP_EN": otp_hex(CONST.HARDENED_FALSE),
+                # By default, the sensor_ctrl should disable all sensors and mark
+                # alerts as recoverable.
+                "OWNER_SW_CFG_ROM_SENSOR_CTRL_ALERT_CFG": [
```

Comment:
Can you also add this to the `sival` and `prodc` SKUs?

Reply:
Done.